xbmc/input/InputManager.cpp

```diff
@@ -646,8 +646,9 @@ bool CInputManager::HandleKey(const CKey& key)
       action = m_buttonTranslator->GetAction(iWin, key);
   }
   if (!key.IsAnalogButton())
-    CLog::LogF(LOGDEBUG, "{} pressed, action is {}",
-               m_Keyboard.GetKeyName((int)key.GetButtonCode()), action.GetName());
+    CLog::LogF(LOGDEBUG, "%s pressed, window %d, action is %s",
```

Comment:
All the recent work has moved away from the printf style modifiers. You can just use {} instead of each modifier as in the original code

Reply:
ouch sorry, my code was based on Leia. Didn't pick it up

Reply:
no dramas